### PR TITLE
Pricing functionality

### PR DESCRIPTION
**Pull request by kart-c**
### Changes made in this PR 
- Update price and discount in database
- Calculations of discount on product listing page and cart page
- Calculations of pricing container 

[Code Preview](https://deploy-preview-9--agitated-stonebraker-54a5ee.netlify.app/) 

>Note: Navigating to the wishlist page crashes the application - The card used in the product listing page and wishlist page are the same, since wishlist functionality is not yet added, some variables remain undeclared and throw an error.

### CODE REVIEW

**Review by RahullRawat — APPROVED** [review]
Functionality is working fine.

Just a small suggestion, when the cart is empty you can provide a button so user can easily navigate to products page

**Comment by AnkurChunekar** [comment]
kartik pr
Please check the above bug it is showing NaN for total price

File: src/backend/db/products.js [hunk 1]
```diff
@@ -97,28 +96,27 @@ export const products = [
...
 		_id: uuid(),
 		title: 'Age of Empires 2',
 		badge: '',
 		image: AOE2,
-		price: 1099,
+		price: 899,
 		rating: 3.5,
-		wishlist: false,
 		categoryName: 'Strategy',
 	},
```
Comment (AnkurChunekar) [inline src/backend/db/products.js]: Only when I add this product it is showing a NaN value, here it doesn't have a discount property, maybe this is the reason why it is showing NaN for the total price value.

File: src/backend/db/products.js [hunk 2]
```diff
@@ -148,7 +146,6 @@ export const products = [
 		image: mario,
 		price: 1399,
 		rating: 4,
-		wishlist: false,
 		categoryName: 'Sports',
 	},
```
Comment (AnkurChunekar) [inline src/backend/db/products.js:152]: check for this product also

File: src/backend/db/products.js (hunk 1 above)
Comment (kart-c) [reply src/backend/db/products.js]: Thank you finding out the bug, I will fix it right away.

File: src/Pages/Cart/Components/Price Container/PriceContainer.jsx [hunk 3]
```diff
@@ -1,22 +1,47 @@
...
 					<span className={styles.amountSpan}>Total Amount</span>
 				</div>
 				<div>
-					<span>5988 /-</span>
-					<span>- 2390 /-</span>
-					<span>40 /-</span>
-					<span className={styles.amountSpan}>3588 /-</span>
+					<span>{totalPrice.toFixed(2)} /-</span>
+					<span>- {discount.toFixed(2)} /-</span>
+					<span>{(40).toFixed(2)} /-</span>
```
Comment (AnkurChunekar) [inline src/Pages/Cart/Components/Price Container/PriceContainer.jsx]: you can remove the `toFIxed()` function from line no. 43

Comment (kart-c) [reply src/Pages/Cart/Components/Price Container/PriceContainer.jsx]: Thank you for pointing it out 👍 fixed it

**Comment by kart-c** [comment]
> Functionality is working fine.
> 
> Just a small suggestion, when the cart is empty you can provide a button so user can easily navigate to products page

Thanks I added the link to products when cart is empty

**Comment by AnkurChunekar** [comment]
- All the functionalities work fine now.
- code is very neat and clean 
Great work @kart-c

**Review by AnkurChunekar — APPROVED** [review]

**Pull request merged**